packages/flutter/lib/src/material/list_tile.dart

```diff
@@ -17,6 +17,18 @@ import 'material_state.dart';
 import 'theme.dart';
 import 'theme_data.dart';
 
+/// Define some default values for convenient external use.
+class ListTileDefaultValue {
```

Comment:
While I like the idea here, it doesn't match the pattern used elsewhere in the code base. To be consistent, can you just use these values directly in the constructor and remove this public class?

Reply:
Moved, and now it is directly in the constructor.